# Group names and their samples
- "HG005": HG005
- "Soma": HG002, HG02622,HG02486, HG02257, HG00438
- "HPRC_non-HapMap": all other samples from the graph, except for CHM13


# Normalize variants from graph vcf (and sort and index)

Script: `/storage2/fs1/epigenome/Active/johnegarza/SMAHT/shared_alleles/bcftools.sh`

This contains two commands, used to create two different versions of the normalized vcf.
Only one was used, generated by the first version of the command; relevant part copied below:

```
bcftools norm \
   --output-type u \
   -m - \
   /storage2/fs1/epigenome/Active/x.zilan/new_graph/HPRC_v1.1_w_hapmap/hprc-v1.1-mc_w_hapmap.vcf.gz \
  | bcftools sort -m 90G -T /scratch1/fs1/epigenome/johnegarza/SMAHT/bcftools-XXXXXX \
   --write-index -o sorted-normed-hprc-v1.1-mc_w_hapmap.vcf.gz -
```


# Count variants

Script: `/storage2/fs1/epigenome/Active/johnegarza/SMAHT/shared_alleles/count2.py`

Exact command:

```
$ bsub -Is -G compute-epigenome -q epigenome-interactive -M 150G -R "select[mem>16G && tmp>150G] rusage[mem=150G]" -R gpuhost -a 'docker(clinicalgenomics/python3.8-cyvcf2-venv:1.0)' bash

$ source /venv/bin/activate

$ python count2.py --vcf sorted-normed-hprc-v1.1-mc_w_hapmap.vcf.gz
Done: VCF subsets and counts.tsv generated.
```

Output file used to generate venn diagrams: `/storage2/fs1/epigenome/Active/johnegarza/SMAHT/shared_alleles/counts.tsv`